pkg/pg/onefieldresult.go

```diff
@@ -80,17 +85,20 @@ func (ofr Result) String() (s string) {
 	return fmt.Sprintf("{ %s }", strings.Join(results, ", "))
 }
 
+// Columns is a function that creates a column of all the results
 func (ofr Result) Columns() (result []string) {
 	for key := range ofr {
 		result = append(result, key)
 	}
 	return result
 }
 
+// FormattedString is a function that formats strings by replacing ' with \\
```

Comment:
FormattedString returns a SQL version of the string.
The string is inserted in single quotes and single quotes in teh string are replaced by 2 single quotes.